### PR TITLE
refactor filemap

### PR DESCRIPTION
**Pull request by YinYangOfDao**
made filemap a modulized yaml file

### CODE REVIEW

File: src/ClusterBootstrap/cloud_init_deploy.py [hunk 1]
```diff
@@ -15,6 +15,8 @@
 from DockerUtils import push_one_docker, build_dockers, push_dockers, run_docker, find_dockers, build_docker_fullname, copy_from_docker_image, configuration
 from params import default_config_parameters
 
+CLD_INIT_FILE_MAP = "cloudinit/file_map.yaml"
```
Comment (Anbang-Hu) [inline src/ClusterBootstrap/cloud_init_deploy.py]: Please try to use full name whenever possible. `CLD` doesn't unique identity word `CLOUD`.

File: src/ClusterBootstrap/cloud_init_deploy.py [hunk 2]
```diff
@@ -530,9 +530,13 @@ def render_worker_node_specific(config, args):
     worker_name = config["worker_node"][0].split(".")[0]
     common_worker_labels = get_kube_labels_of_machine_name(config, worker_name)
     config = get_stat_of_sku(config)
+    default_worker_f2cp = ["kubernetes_common", "kubelet_worker", "nfs_client"]
     for sku in config["worker_sku_cnt"]:
-        config["kube_labels"] = common_worker_labels + ["sku={}".format(sku), "gpuType={}".format(
-            config.get("sku_mapping", {}).get(sku, {}).get("gpu-type", "None"))]
+        gpuType = config.get("sku_mapping", {}).get(sku, {}).get("gpu-type", "None")
```
Comment (Anbang-Hu) [inline src/ClusterBootstrap/cloud_init_deploy.py]: python variable should be snake-cased: `gpu_type`

File: src/ClusterBootstrap/deploy.sh [hunk 3]
```diff
@@ -11,6 +11,8 @@ rm -rf deploy/* cloudinit* az_complementary.yaml
 ./cloud_init_deploy.py docker servicesprerequisite
 # generate scripts
 ./cloud_init_aztools.py -v deploy
+# ./cloud_init_aztools.py -cnf config.yaml -cnf az_complementary.yaml -v --dryrun --output scripts/framework.sh deployframework
```
Comment (Anbang-Hu) [inline src/ClusterBootstrap/deploy.sh]: Remove unused code.

File: src/ClusterBootstrap/cloud_init_deploy.py (hunk 2 above)
Comment (YinYangOfDao) [reply src/ClusterBootstrap/cloud_init_deploy.py]: this is kind of legacy code. should we change all "gpuType" and "gpu-type" to "gpu_type"?

Comment (Anbang-Hu) [reply src/ClusterBootstrap/cloud_init_deploy.py]: no need. I was talking about the python variable, not the variable names in configuration.

**Pull request merged**